nautobot/core/templates/base.html

```diff
@@ -29,15 +32,13 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>{{ settings.BRANDING_TITLE }}</title>
-    <script defer src="/static/js/bundle.js"></script>
+    <script defer src="{% static 'js/main.js' %}"></script>
+    <link href="{% static 'css/main.css' %}" rel="stylesheet">
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
     <div id="legacy-content" style="display:none">
-      {% load static %}
-      {% load helpers %}
-      {% load plugins %}
```

Comment:
This (and the line below it) allows us to still render tab views for objects. I know we discussed forcing the tab views to be implemented without being complete views but knowing that this will prevent the block below from rendering content.